src/main.ts

```diff
@@ -2,7 +2,6 @@ import * as nacl from './nacl/naclWrappers';
 import * as address from './encoding/address';
 import * as encoding from './encoding/encoding';
 import * as txnBuilder from './transaction';
-import * as LogicTemplatesCommonJSExport from './logicTemplates';
```

Comment:
Unfortunately we can't remove the template exports without it being a breaking change -- this is akin to deleting the templates.